src/common/balances.lua

```diff
@@ -1,54 +1,81 @@
+--- Module for managing balances and transactions.
+-- @module balances
+
 local utils = require(".common.utils")
-local json = require(".common.json")
 
 local balances = {}
 
-function balances.walletHasSufficientBalance(wallet)
-	return Balances[wallet] ~= nil and Balances[wallet] > 0
-end
-
+--- Transfers the ANT to a specified wallet.
+---@param to string - The wallet address to transfer the balance to.
+---@return nil
 function balances.transfer(to)
 	utils.validateArweaveId(to)
 	Balances = { [to] = 1 }
+	--luacheck: ignore Owner Controllers
 	Owner = to
 	Controllers = {}
-	return json.encode({ [to] = 1 })
 end
```

Comment:
nit - i would still return the data here, just not wrapped in encode

Reply:
[9d514a3](https://github.com/ar-io/ar-io-ant-process/pull/32/commits/9d514a31d5b973b5918b287e64ae8da2909ce93e)